VK/VK/Services/Models/FeedModels/PurpleLikes.swift

```diff
@@ -0,0 +1,13 @@
+// PurpleLikes.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Лайки
```

Comment:
```suggestion
/// Лайки
```
тут собаки 
и тут собаки struct FluffyLikes: Decodable {